search/tutorials/tutorial_ordering.md

```diff
@@ -1,136 +1,216 @@
-```bash
-pip install virtualenv
-```
-```bash
-virtualenv <your-env>
-```
-```bash
-source <your-env>/bin/activate
-```
-Next, install Google packages:
-```bash
-pip install google
-```
-```bash
-pip install google-cloud-retail
-```
+### Select your project and enable the Retail API
 
-**Tip**: Click the copy button beside the code box to paste the command in the Cloud Shell terminal and run it.
+Google Cloud organizes resources into projects. This lets you
+collect all the related resources for a single application in one place.
 
-## Set the PROJECT_NUMBER environment variable
+If you don't have a Google Cloud project yet, you can
+[create a new project](https://console.cloud.google.com/projectcreate).
 
-As you are going to run the code samples in your own Cloud Project, you should specify the **project_id** as an environment variable, it will be used in every request to the Retail API.
+After the project is created, set your PROJECT_ID to a ```project``` variable.
+1. Run the following command in Terminal:
+    ```bash
+    gcloud config set project <YOUR_PROJECT_ID>
+    ```
 
-You can find the ```project_number``` in the **Home/Dashboard/Project Info card**.
+1. Check that the Retail API is enabled for your Project in the [Admine Console](https://console.cloud.google.com/ai/retail/).
 
-Set the environment variable with a following command:
-```bash
-export PROJECT_NUMBER=<YOUR_PROJECT_NUMBER>
-```
+### Set up authentication
 
-## Ordering by a single field. Ordering expression
+To run a code sample from the Cloud Shell, you need to authenticate. To do this, use the Application Default Credentials.
 
-To use the ordering feature, you need to specify the field and the ordering direction. You can order by both the text and numeric fields.
+1. Set your user credentials to authenticate your requests to the Retail API
 
-First, let's order the search results by price when more expensive items come first. To do that, set the ordering expression as follows: 
+    ```bash
+    gcloud auth application-default login
+    ```
 
-```order = 'price desc'```
- 
+1. Type `Y` and press **Enter**. Click the link in Terminal. A browser window should appear asking you to log in using your Gmail account.
 
-To see the whole request with ordering, open **search_with_ordering.py**
+1. Provide the Google Auth Library with access to your credentials and paste the code from the browser to the Terminal.
 
-Run the sample in a terminal using the following command:
-```bash
-python search_with_ordering.py
-```
+1. Run the code sample and check the Retail API in action.
+
+**Note**: Click the copy button on the side of the code box to paste the command in the Cloud Shell terminal and run it.
+
+### Set the PROJECT_NUMBER environment variable
+
+Because you are going to run the code samples in your own Google Cloud project, you should specify the **project_number** as an environment variable. It will be used in every request to the Retail API.
```

Comment:
In line 26, we use PROJECT_ID, and here we use PROJECT_NUMBER. Are they the same?